dts/arm/gigadevice/gd32e10x/gd32e10x.dtsi

```diff
@@ -460,6 +460,28 @@
 			};
 		};
 
+		dma0: dma@40020000 {
+			compatible = "gd,gd32-dma";
+			reg = <0x40020000 0x400>;
+			interrupts = <11 0>, <12 0>, <13 0>, <14 0>, <15 0>, <16 0>, <17 0>;
+			rcu-periph-clock = <0x500>;
+			dma-channels = <7>;
+			#dma-cells = <1>;
+			status = "disabled";
+			label = "DMA_0";
```

Comment:
@soburi and @cameled ,  Let's not introduce anymore label properties. Currently all properties will be dropped, see #48014 as reference.

Reply:
Remove label from all dtsi files.

Reply:
Update tests/drivers/dma/loopback_transfer following new manner that show in #48276.
Not only dts.